app/build.gradle

```diff
@@ -28,7 +28,11 @@ repositories {
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
-  implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
+
+  //downgraded version for design support library
```

Comment:
Use the beta version from here: https://developer.android.com/topic/libraries/support-library/revisions

Reply:
:ok: